solarwinds_apm/configurator.py

```diff
@@ -313,13 +313,15 @@ def _configure_traces_exporter(
         apm_fwkv_manager: SolarWindsFrameworkKvManager,
         apm_config: SolarWindsApmConfig,
     ) -> None:
-        """Configure SolarWinds OTel span exporters, defaults or environment
-        configured, or none if agent disabled.
+        """Configure traces exporters if agent enabled.
+        Links to global TracerProvider.
 
         Initialization of SolarWinds exporter requires a liboboe reporter
-        Note: if reporter is no-op, the SW exporter will not export spans."""
+        If `reporter` is no-op, the SW exporter will not export spans."""
         if not apm_config.agent_enabled:
-            logger.error("Tracing disabled. Cannot set trace exporter.")
+            logger.error(
+                "APM Python library disabled. Cannot set traces exporters."
```

Comment:
Not changing `_configure_traces_exporter`, just updating docstring and error log for consistency.